### ISSUE TITLE
任務ウィンドウの右クリックによる検索の仕様の要望

### ISSUE THREAD

**Issue by KimidoriBoushi** [gre4bee/ElectronicObserver#126]
現仕様において、DuckDuckGoとなっておりますが、この検索エンジンを、どこにするか選択式になったらたいへん助かるのですがご検討願えませんでしょうか。

具体的には、Googleが一番都合がいいのです。

理由としましては、私は攻略サイトは「ぜまかしねっと艦これ」派なのですが、DuckDuckGoですと出てこない事があるのです。

大変身勝手な要望ですが、検索エンジンを選べる仕様を検討願えませんでしょうか。

**Comment by myangelkamikaze** [comment]
Yeah, it was mentioned before that DuckDuckGo can't find Zekamashi for some reason. We'll look into what the best thing to do here is.
